ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -110,6 +112,12 @@ private static void createBuses(UcteNetworkExt ucteNetwork, UcteVoltageLevel uct
                 continue;
             }
 
+            reporter.report(Report.builder()
+                .withKey("createBus")
+                .withDefaultMessage("Create bus ${bus}")
+                .withValue("bus", ucteNodeCode.toString())
+                .withSeverity(Report.SEVERITY_TRACE)
+                .build());
```

Comment:
I would have removed the equivalent Logger.trace and configured the LoggerReport (with allowing the dynamically specify the category) by default. Also allow to pass multiple reporter (or give an api to create a kind of DispatchingReporter..). Did you plan to do this in the future ?

Reply:
I didn't plan it so far, as no usecase is identified, but passing multiple reporters could be done in the future, through a compound implementation as @sylvlecl suggested.